src/App.tsx

```diff
@@ -1,24 +1,159 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import { Todo } from './types/Todo';
+import { deleteTodo, getTodos, postTodo } from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { ErrorNotification } from './components/ErrorNotification';
+import { LoadError } from './types/LoadError';
+import { FilterType } from './Enums/FilterType';
+import { filterTodos } from './utils/filterTodos';
+import { NewTodoForm } from './components/NewTodoForm';
+import { PostTodo } from './types/PostTodo';
 
-const USER_ID = 0;
+export const USER_ID = 10895;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterType, setFilterType] = useState(FilterType.ALL);
+  const [loadError, setError] = useState<LoadError>({
+    status: false,
+    message: '',
+  });
```

Comment:
Hey, well done! 🔥 

I have only one suggestion for you.

I think it will be more convenient to create a separate `ENUM` with all the error messages and use them.
It will minimize the code a bit)

Reply:
Interesting, thanks!